en/components/charts/types/column-chart.md

```diff
@@ -81,7 +81,7 @@ The Column Chart is able to render multiple columns per category for comparison
 
 The Angular Column Chart has many options for styling and modification of the visual appearance.
 
-You can create this type of chart in the [`IgxDataChartComponent`]({environment:dvApiBaseUrl}/products/ignite-ui-angular/api/docs/typescript/latest/classes/igniteui_angular_charts.igxdatachartcomponent.html) control by binding your data to a [`IgxColumnSeriesComponent`]({environment:dvApiBaseUrl}/products/ignite-ui-angular/api/docs/typescript/latest/classes/igniteui_angular_charts.igxcolumnseriescomponent.html), as shown in the example below:
+You can create this type of chart in the `XamCategoryChart` control by binding your data, as shown in the example below:
```

Comment:
I think this change is faulty. We don't have any replacement logic to change XAM to igx in Angular right?

Reply:
ok we will correct it